views/Signup.js

```diff
@@ -1,8 +1,212 @@
+            isValid,
+          }) => (
+            <View>
+              <TextInput
+                name="username"
+                placeholder="Username"
+                onChangeText={handleChange('username')}
+                onBlur={handleBlur('username')}
+                value={values.username}
+                style={styles.textInput}
+              />
+              {errors.username && touched.username && (
+                <Text style={{ fontSize: 10, color: 'red' }}>
+                  {errors.username}
+                </Text>
+              )}
+              <TextInput
+                type="email"
+                name="email"
+                placeholder="Email"
+                onChangeText={handleChange('email')}
+                onBlur={handleBlur('email')}
+                value={values.email}
+                style={styles.textInput}
+              />
+              {errors.email && touched.email && (
+                <Text style={{ fontSize: 10, color: 'red' }}>
+                  {errors.email}
+                </Text>
+              )}
+              <TextInput
+                type="password"
+                name="password"
+                placeholder="Password"
+                secureTextEntry
+                onChangeText={handleChange('password')}
+                onBlur={handleBlur('password')}
+                value={values.password}
+                style={styles.textInput}
+              />
+              {errors.password && touched.password && (
+                <Text style={{ fontSize: 10, color: 'red' }}>
+                  {errors.password}
+                </Text>
+              )}
+              <TextInput
+                type="confirmPassword"
+                name="confirmPassword"
+                placeholder="Confirm password"
+                secureTextEntry
+                onChangeText={handleChange('confirmPassword')}
+                onBlur={handleBlur('confirmPassword')}
+                value={values.confirmPassword}
+                style={styles.textInput}
+              />
+              {errors.confirmPassword && touched.confirmPassword && (
+                <Text style={{ fontSize: 10, color: 'red' }}>
+                  {errors.confirmPassword}
+                </Text>
+              )}
+              <TextInput
+                type="full_name"
+                name="full_name"
+                placeholder="Full name"
+                onChangeText={handleChange('full_name')}
+                onBlur={handleBlur('full_name')}
+                value={values.full_name}
+                style={styles.textInput}
+              />
+              {errors.full_name && touched.full_name && (
+                <Text style={{ fontSize: 10, color: 'red' }}>
+                  {errors.full_name}
+                </Text>
+              )}
+              <Button
+                disabled={!isValid}
+                onPress={handleSubmit}
+                title="Submit"
+              />
```

Comment:
Because we remove the header, we can add the link here to come back to sign in page